cfe_internal/enterprise/mission_portal.cf

```diff
@@ -236,6 +247,13 @@ bundle agent mission_portal_apache_from_stage(config, staged_config)
         comment => concat( "We start apache after the new valid config is in ",
                            "place only if we have stopped apache already.");
 
+      "cf-apache" -> { "ENT-9686" }
+        service_policy => "restart",
+        if => and( "mission_portal_apache_config_repaired",
+                   not( "apache_stop_after_new_staged_config_repaired" ) ),
+        comment => concat( "We restart apache after the new valid config is in ",
+                           "place only if we have stopped apache already.");
```

Comment:
Is this part of the comment accurate? If so, why not also on the non-systemd version?

Reply:
Yeah, I don't think this part of the comment is accurate.  In fact, it's the opposite, we restart apache after the new valid config is in place only if we have NOT stopped apache already. Right? Maybe add that here and above as well?